### PR TITLE
Potential fix for splash screen crashes

### PR DESCRIPTION
**Pull request by Caedis**
Found by wohaopa in #github-discussion

### CODE REVIEW

File: src/main/java/com/gtnewhorizons/angelica/hudcaching/HUDCaching.java [hunk 1]
```diff
@@ -187,15 +190,15 @@ public static void fixGLStateBeforeRenderingCache() {
     }
 
     private static void resetFramebuffer(int width, int height) {
-        if (framebuffer.framebufferWidth != width || framebuffer.framebufferHeight != height) {
-        	framebuffer.createBindFramebuffer(width, height);
-            framebuffer.setFramebufferFilter(GL11.GL_NEAREST);
+        if (getFramebuffer().framebufferWidth != width || getFramebuffer().framebufferHeight != height) {
```
Comment (mitchej123) [inline src/main/java/com/gtnewhorizons/angelica/hudcaching/HUDCaching.java:193]: nit: store the FB in a final local variable... it annoys me calling a getter every time.....

**Comment by mitchej123** [comment]
One other thing that comes to mind, any reason not to do a late init on world login so we don't accidentally do HUDCaching until we're in the world?  Similar to how I don't enable the GLSM caching until login....

**Comment by Caedis** [comment]
hold off on merging

**Comment by Caedis** [comment]
Good now

**Review by mitchej123 — APPROVED** [review]

**Comment by Abalieno** [comment]
Overmind in greg discord suggested to try with HUD caching off, and it does indeed removes the issue.

Hopefully this helps tracking it and it can be addressed in some way?

**Pull request merged**